pandas/core/series.py

```diff
@@ -374,6 +374,10 @@ def get_values(self):
         """ same as values (but handles sparseness conversions); is a view """
         return self._data.get_values()
 
+    @property
```

Comment:
@jreback Currently ``Index.asobject`` is defined only in box-able ``Index``. We can add it but internal implementations are different from ``Series``, because ``Index._data`` returns ``ndarray``.